stable/varnish/Makefile

```diff
@@ -0,0 +1,9 @@
+snapshot:
+	rm -f tests/__snapshot__/*
```

Comment:
Shouldnt it be better to put this in the Makefile:
```
helm unittest . --update-snapshot
```

Reply:
I'm migrating the existing, not fixing tech-debt 😉